packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/index.js

```diff
@@ -27,6 +27,12 @@ export default function SidebarNavigationScreenNavigationMenus() {
 					postId: attributes.id,
 				} );
 			}
+			if ( name === 'core/page-list-item' && attributes.id && history ) {
```

Comment:
Should we be adding a "name" check to the first case as well (check that we're clicking on navigation items) for safeties or not?

Reply:
I don't think we need a check if an item is selected and has a post type and id it is safe to assume we should move there. In the future we may have more types of items or even third parties I think just relying on kind and id should be enough.